itsUk45/Property-Pro-Lite-
# #167270354 add functionality and bug fix

**What does this PR do?**
- update user interface module
- fix api route bugs

**Description of Task to be completed**
- update ui.
- improve page navigation.
- resolve delete property api routes bug

**How should this be manually tested?**
- clone the repo and switch to the branch bug-develop-api-mocha-167270354
- run nodemon app to start the server
- run npm run coveralls to see the test output
- navigate to localhost:5000 to start performing api operations
  ex to signup a user navigate to localhost:5000/auth/signup and enter the relevant request body content

**Any background context you want to provide?**
    - N/A

**What are the relevant pivotal tracker stories?**
    #167270354
    **Screenshots (if appropriate)**
    - N/A
